Source/Assets/PlayFabEditorExtensions/Editor/Scripts/Panels/PlayFabEditorSDKTools.cs

```diff
@@ -233,10 +233,15 @@ public static Type GetPlayFabSettings()
                 return playFabSettingsType;
 
             playFabSettingsType = typeof(object); // Sentinel value to indicate that PlayFabSettings doesn't exist
-            foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
+            var allAssemblies = AppDomain.CurrentDomain.GetAssemblies();
+            foreach (var assembly in allAssemblies)
                 foreach (var eachType in assembly.GetTypes())
                     if (eachType.Name == PlayFabEditorHelper.PLAYFAB_SETTINGS_TYPENAME)
                         playFabSettingsType = eachType;
+            //if (playFabSettingsType == typeof(object))
```

Comment:
should this commented code be left in for future un-commenting?

Reply:
Yes. One of us will need to dig deeper here in order to figure out why the PlayFab settings isn't returning from the assemblies a few lines up